Client/Frontend/Intro/IntroViewController.swift

```diff
@@ -45,8 +45,7 @@ struct IntroViewControllerUX {
 let IntroViewControllerSeenProfileKey = "IntroViewControllerSeen"
 
 protocol IntroViewControllerDelegate: class {
-    func introViewControllerDidFinish(_ introViewController: IntroViewController)
-    func introViewControllerDidRequestToLogin(_ introViewController: IntroViewController)
+    func introViewControllerDidFinish(_ introViewController: IntroViewController, requestToLogin: Bool)
```

Comment:
With the power vested in me I merge this Protocol into 1 function! 🕺

Reply:
👍